### PR TITLE
SW course sections can now start with letters, not only digits

### PR DESCRIPTION
**Pull request by mrenoch**
SW reported problems adding sections keys with leading letters (not digits) to care-ssw (aka carr).  I think this will likely affect our other apps too. Would be nice if we could learn about such changes in rules, but cest la vie. 

I added a test and changed the regex here, though after someone looks at this it probably needs to be redeployed to at least carr and medithread. Not sure what else.

### CODE REVIEW

File: courseaffils/columbia.py [hunk 1]
```diff
@@ -145,7 +145,7 @@ def to_string(cdict):
     @staticmethod
     def to_dict(wind_string):
         wind_match = re.match(
-            ('t(?P<term>\d).y(?P<year>\d{4}).s(?P<section>\d{3})'
+            ('t(?P<term>\d).y(?P<year>\d{4}).s(?P<section>\w{3})'
```
Comment (thraxil) [inline courseaffils/columbia.py]: This change is in the WindTemplate regexp, but the test you added is for the SectionkeyTemplate. I'm not sure which one you actually intended to change.

Comment (mrenoch) [inline courseaffils/columbia.py]: Hmm... I think I actually need to change this in all three occurrences of the regex then. Somehow, when I changed this one, my test started passing, but it does seem like the new patter is now valid, so I think it needs to be updated everywhere.

**Comment by mrenoch** [comment]
I am closing this PR for now. Valid section keys still need to be updated, but carr has it's own logic.

File: courseaffils/columbia.py (hunk 1 above)
Comment (sdreher) [inline courseaffils/columbia.py]: * WindTemplate test added.
* SectionKeyTemplate updated to accept alphanumeric section keys.

**Pull request merged**